Jupyter Notebooks/LUVOIR/asahoo_merging_1.ipynb

```diff
@@ -0,0 +1,502 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eight-turtle",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pwd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "alike-tablet",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.colors import LogNorm\n",
+    "from astropy.io import fits\n",
+    "import astropy.units as u\n",
+    "import hcipy as hc\n",
+    "import matplotlib as mpl\n",
+    "os.chdir('/Users/asahoo/repos/PASTIS/')\n",
```

Comment:
You won't need this line anymore. This was necessary in a previous version of the package, before it was an actual package.